### PR TITLE
Unicode pieces

### PR DESCRIPTION
**Pull request by ce8244a**
- Add color method to piece model
- Add unicode to bishop, king, knight, pawn, piece, queen, and rook model.

### CODE REVIEW

**Comment by hoitomt** [comment]
Make sure to do a git pull before you fix any thing.

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -12,8 +12,17 @@
...
       <td>
-          #code goes here
+          <% def piece_by_position(game, x, y)
+            piece = game.piece_at(x,y)
+            if piece.nil?
+              return ""
+            else
+            return piece.to_ascii_piece
+            end
+          end %>
```
Comment (hoitomt) [inline app/views/games/show.html.erb]: Let's do something different here - you have a method inserted into a view.

How about this:

```
<td>
  <% piece = Piece.find_by_position(x, y) %>
  <% if piece.present? %>
    <%= piece.to_ascii_piece %>
  <% end %>
</td>
```

**Comment by hoitomt** [comment]
Hi @ce8244a - Please fix my one last comment regarding the method in the view and then we can merge this PR. Make sure to do a `git pull` before fixing anything. I pushed a whitespace fix that you need to pull down first.

**Comment by ce8244a** [comment]
- Revised code on show.html page
- Removed turbolinks in app/assets/javascript/application.js - line 16

**Pull request merged**